.github/workflows/ci.yml

```diff
@@ -272,3 +280,58 @@ jobs:
           files: |
             packages/*
             LICENSE
+  docker-image:
```

Comment:
Please change entire job according to https://github.com/Itsusinn/tuic/blob/dev/.github%2Fworkflows%2Fci.yml#L250-L305